frontend/src/container/OnboardingContainer/Steps/ConnectionStatus/ConnectionStatus.tsx

```diff
@@ -57,8 +64,64 @@ export default function ConnectionStatus(): JSX.Element {
 		maxTime,
 		selectedTime,
 		selectedTags,
+		options: {
+			enabled: getStartedSource !== 'kafka',
+		},
+	});
+
+	const [pollInterval, setPollInterval] = useState<number | false>(10000);
+	const {
+		data: onbData,
```

Comment:
onbData ?
onbErr ?

Reply:
+1

Reply:
changed to - `onboardingStatusData`, `onboardinfStatusError` & `onboardingStatusFetching`